tests/unit/lms/services/rsa_key_service_test.py

```diff
@@ -12,6 +12,20 @@
 
 
 class TestAESService:
+    def test_rotate(self, svc_generates, valid_keys, expired_keys, db_session):
+        target_keys = 5
+
+        # Assert what we think is the initial setup
+        assert not any((key.expired for key in valid_keys))
+        assert all((key.expired for key in expired_keys))
+        assert all((key in db_session for key in expired_keys))
+
+        svc_generates.rotate(target_keys)
+
+        assert all((key.expired for key in valid_keys))
```

Comment:
Not asserting anything about the keys themselves (other than the expiration status)